react-hot-button/my-app/src/button.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import './button.css';
+
+export function Button() {
+  const [index, setIndex] = useState(0);
+  let color = 'purple';
```

Comment:
You don't need to actually give this variable an initial value. You could just say, `let color;`.